EasyPost/Models/API/Address.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EasyPost._base;
+using EasyPost.Utilities.Annotations;
+using Newtonsoft.Json;
+using RestSharp;
+
+namespace EasyPost.Models.API
+{
+    public class Address : EasyPostObject
+    {
+        #region JSON Properties
+
+        [JsonProperty("carrier_facility")]
+        public string carrier_facility { get; set; }
+
+        [JsonProperty("city")]
+        public string city { get; set; }
+        [JsonProperty("company")]
+        public string company { get; set; }
+        [JsonProperty("country")]
+        public string country { get; set; }
+        [JsonProperty("email")]
+        public string email { get; set; }
+        [JsonProperty("error")]
+        public string error { get; set; }
+        [JsonProperty("federal_tax_id")]
+        public string federal_tax_id { get; set; }
+        [JsonProperty("message")]
+        public string message { get; set; }
+        [JsonProperty("mode")]
+        public new string mode { get; set; }
```

Comment:
Question: Does `EasyPostObject` have `mode` and all of that now? If so should we remove it here?

Reply:
Good catch. Resolved.